frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
```

Comment:
This declaration doesn't work.
<img width="473" alt="Screenshot 2022-08-02 at 08 48 55" src="https://user-images.githubusercontent.com/26620750/182310149-1d9e1b82-2e39-493a-bc32-3928626cc9a4.png">

Reply:
I changed to:
```
box-shadow: 0 4px 5px #9f18d9;
opacity: 0.5;
```